### PR TITLE
+8 New icons: Text Formatting, Arrows and Pain Tool categories

### PR DESCRIPTION
**Pull request by HubertRyanOfficial**
### ✨ Description of New Features

1. ➡️⬅️ Left/Right Arrow Icons: Added icons for navigating left and right.
2. 📝 Text and Formatting Icons: Added new icons for text formatting functionalities, such as: bold, italic, underline and at-sign. 
3. 🎨 Paint Tool Icon: Added a new icon to represent the paint tool.

https://github.com/user-attachments/assets/cbc03536-12b5-4aa4-b4a6-e5dd1c2124fe

These additions expand the icon set with useful options for navigation, text editing, and design tools. All icons are animated and ready for use!

_by Hubert Ryan_

### CODE REVIEW

File: yarn.lock
Comment (pqoqubbw) [inline yarn.lock:1]: could you please revert this change? 👀

**Comment by pqoqubbw** [comment]
hey, @HubertRyanOfficial , thank you for all of these icons! that is great work. i love the pen-tool, arrows and bold one

italic and underline are also great, could you make them a little faster tho?
at-sign and undo, i guess, we need to rethink

i'll merge it after your fixes and if you don't mind, i'll update the at-sign and undo cause i've got an idea

**Comment by HubertRyanOfficial** [comment]
> hey, @HubertRyanOfficial , thank you for all of these icons! that is great work. i love the pen-tool, arrows and bold one
> 
> italic and underline are also great, could you make them a little faster tho? at-sign and undo, i guess, we need to rethink
> 
> i'll merge it after your fixes and if you don't mind, i'll update the at-sign and undo cause i've got an idea

Hey there, it's a pleasure.  I've been looking for good open source projects to contribuite and I found yours, so it's perfect.

Changes that you asked me are **done**. ✅
1. Italic and underliner faster.
2. Yarn.lock is back.
